apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -353,37 +359,50 @@ class ReviewTab extends Component {
       projectOwnerName
     } = this.state;
 
-    return (
-      <div style={styles.reviewsContainer}>
-        <div style={styles.reviewHeader}>
-          {viewAs !== ViewType.Teacher &&
-            !errorLoadingReviewblePeers &&
-            (viewAsCodeReviewer
-              ? this.renderBackToMyProject(this.onClickBackToProject)
-              : this.renderPeerDropdown(reviewablePeers, this.onSelectPeer))}
-          {this.renderReadyForReviewCheckbox()}
+    // channelId is not available on projects where the student has not edited the starter code.
```

Comment:
Git is picking this up as a diff, but it just puts the previous return statement inside of an else block.